config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml

```diff
@@ -0,0 +1,71 @@
+periodics:
+  - name: soak-tests-capz-windows-2019
+    interval: 1w
+    decorate: true
+    decoration_config:
+      timeout: 8h
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-azure-cred-only: "true"
+      preset-azure-anonymous-pull: "true"
+      preset-azure-windows: "true"
+    extra_refs:
+      #- org: kubernetes
+      - org: oprinmarius
+        repo: perf-tests
+       # base_ref: main
+        base_ref: "windows-soak-tests"
+        path_alias: "sigs.k8s.io/perf-tests"
+      - org: kubernetes-sigs
+        repo: cluster-api-provider-azure
+        base_ref: main
+        path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+        workdir: true
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220420-a8dc19ed91-master
+          command:
+            - runner.sh
+            - ./scripts/ci-entrypoint.sh
+          args:
+            - bash
+            - -c
+            - >-
+              cd ${GOPATH}/src/sigs.k8s.io/perf-tests/clusterloader2 &&
+              go run cmd/clusterloader.go 
+              --testconfig=testing/windows-tests/config.yaml
+              --provider=skeleton 
+              --report-dir=${REPORTS_DIR} 
+              --v=2 
+          securityContext:
+            privileged: true
+          env:
+            - name: WINDOWS
```

Comment:
for these we can use the preset already defined like https://github.com/kubernetes/test-infra/pull/25604/files